karma.conf.cjs

```diff
@@ -86,7 +88,7 @@ module.exports = function (config) {
 
     html2JsPreprocessor: {
       // strip this from the file path
-      stripPrefix: 'test/',
+      stripPrefix: 'test/unit/',
```

Comment:
Do you also need to strip `data-driven/`?

Reply:
This is a convenience option we use in the unit tests so we load html files without having to specify the full path. In the case of data driven, the test infra is configured with paths for input, output and html files so we don't really need it.

Reply:
Sounds good, thanks for the clarification